modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/components/StoryGradientBackground.kt

```diff
@@ -32,12 +37,14 @@ fun StoryBlurredBackground(
                     x = offset.x.toInt().pxToDp(context).dp,
                     y = offset.y.toInt().pxToDp(context).dp
                 )
+                .rotate(rotate)
         ) {
             Image(
                 painterResource(id = R.drawable.story_blurred_background),
                 contentDescription = null,
                 contentScale = ContentScale.Crop,
                 modifier = Modifier.fillMaxSize()
+                    .alpha(0.6f)
```

Comment:
I added alpha to the blurred background as those looked brighter than Figma backgrounds. This has also improved the readability of the title on the "Top Podcast" story. Let me know if you have other ideas.

Before | After
-----|-----
<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/041043c1-f9e9-4f0c-a792-145b06b8feec"/> | <img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/b4af661b-1418-404f-92f4-5c8d2c4a606c"/>